pkg/converter/tool/builder.go

```diff
@@ -7,13 +7,17 @@
 package tool
 
 import (
+	"context"
 	"fmt"
 	"os/exec"
 	"strings"
+	"time"
 
 	"github.com/sirupsen/logrus"
 )
 
+const maxTimeOut = 2 * time.Hour
+
 var logger = logrus.WithField("module", "builder")
 
 type PackOption struct {
```

Comment:
Can we define a `Timeout *time.Duration` field in `PackOption/MergeOption/UnpackOption`? `nil` means no timeout.